docs/source/api/v4/logs.rst

```diff
@@ -37,7 +37,7 @@ Request Structure
 	+===========+==========+=====================================================================================================================================+
 	| days      | no       | An integer number of days of change logs to return                                                                                  |
 	+-----------+----------+-------------------------------------------------------------------------------------------------------------------------------------+
-	| limit     | no       | The number of records to which to limit the response                                                                                |
+	| limit     | no       | The number of records to which to limit the response, by default it will display all the response                                                                               |
```

Comment:
This table is now malformed, needs to be aligned - but also, you don't need to make mention of there not being a default, instead the older API versions should make it clear that there _is_ a default. One would expect that if a limit is not given, then there is no limit. The docs should just clarify when those expectations are not met.

Reply:
Fixed the malformed tables.